src/providers/AppState.js

```diff
@@ -20,26 +22,50 @@ function AppStateProvider({ children }) {
     installedApps,
     requestToken,
     stakeToken,
+    totalStaked,
+    minThresholdStakePercentage,
+    pctBase,
     ...appData
   } = useAppData(organization)
 
   const vaultBalance = useVaultBalance(installedApps, requestToken)
+
   const { balance, totalSupply } = useTokenBalances(account, stakeToken)
+
+  const effectiveSupply = useMemo(() => {
+    if (
+      !(totalSupply && totalStaked && minThresholdStakePercentage && pctBase)
+    ) {
+      return
+    }
+
+    const percentageOfTotalSupply = totalSupply
+      .multipliedBy(minThresholdStakePercentage)
+      .div(bigNum('1'))
```

Comment:
This is `10^18` as this is a helper function. The contract uses this value to do this math, [as defined here](https://github.com/1Hive/conviction-voting-app/blob/c9f9325fb4fb10af439fbc03eaa7f666c0439bf3/contracts/ConvictionVoting.sol#L22).